policy/modules/services/ssh.if

```diff
@@ -174,6 +174,14 @@ template(`ssh_server_template', `
 		attribute ssh_server;
 		type sshd_exec_t, sshd_key_t;
 	')
+
+	## <desc>
+	## <p>
+	##	Allow sshd to use remote port forwarding (bind to any TCP port)
+	## </p>
+	## </desc>
+	gen_tunable($1_use_remote_port_forwarding, false)
```

Comment:
Let's go a bit simpler for the name; how about `$1_port_forwarding`?

Reply:
Definitely much simpler. Let's use that.